big-data-utils/README.md

```diff
@@ -127,3 +127,11 @@ You can easily call commonly used Lucene classes from the static utility library
 	}
 ```
 
+## Citation
+
+Thank you for citing us if you use us in your work!
+
```

Comment:
For all the places where you could not find an actual paper to cite, you can add the following text: 

If you use this code in your research, please provide the URL for this 
github repository in the relevant publications.